dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -88,7 +88,12 @@
     {
         "TUNNEL_DECAP_TABLE:IPINIP_TUNNEL" : {
             "tunnel_type":"IPINIP",
+{% if "broadcom" in asic_type %}
```

Comment:
Please link the sonic-mgmt PR to description. Also how about subnet decap. Can you scan for `dscp_mode `in the file and address this? @lolyu for viz